### PR TITLE
two minor adjustments (0-RTT and "might have been")

### PR DESCRIPTION
**Pull request by wtarreau**
Found while reading the latest version.

### CODE REVIEW

File: draft-thomson-http-replay.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ in early data does not need to include the `Early-Data` header field.
 ## The 4NN (Too Early) Status Code {#status}
 
 A 4NN (Too Early) status code indicates that the server is unwilling to risk
-processing a request that might be (or has been) replayed.
```
Comment (martinthomson) [inline draft-thomson-http-replay.md:226]: I was planning to make this "might be or might have been".

**Comment by martinthomson** [comment]
I cherry-picked the first commit already :)

File: draft-thomson-http-replay.md (hunk 1 above)
Comment (wtarreau) [reply draft-thomson-http-replay.md:226]: oh yes, even better!

**Comment by mnot** [comment]
OBE.

**Pull request closed without merging**